dodany teamModal + delikatnie zmieniony styl modala

Marcinn88 opened the pull request:
Aby zobaczyc modal w akcji trzeba najechac na napis wallet i kliknac . Modal dziala tylko na stronie do logowania.

Marcinn88 committed:
dodany teamModal + delikatnie zmieniony styl

Aby zobaczyc modal w akcji trzeba najechac na napis wallet i kliknac . Modal dziala tylko na stronie do logowania.

MateuszCharysz reviewed (changes requested):
coś jest nie teges ... jak otwiera to obcina parę osób

Marcinn88 committed:
fix

Sprawdz czy teraz bedzie Ci ucinac. Jak tak to znaczy ze zmieniono inne style. W takim przypadku poprawie to juz na gotowo.

MateuszCharysz reviewed (approved):
ok teraz jest lepsze w odbiorze
